web/public/css/style.css

```diff
@@ -0,0 +1,88 @@
+@import url('https://fonts.googleapis.com/css?family=Dosis|Roboto');
```

Comment:
Move the fonts to the `<head>`, this is blocking, see Google Fonts for instructions. Or maybe figure out a way to do async font loading?